apps/server/src/modules/authentication/mapper/current-user.mapper.ts

```diff
@@ -38,6 +45,7 @@ export class CurrentUserMapper {
 			schoolId: jwtPayload.schoolId,
 			userId: jwtPayload.userId,
 			impersonated: jwtPayload.support,
+			isExternalUser: false,
```

Comment:
The mapping from jwtPayload to ICurrentUser is missing.

There should also be a mapping between ICurrentUser and CreateJwtPayload in the login controller that is missing. This could be problematic since ICurrentUser has the "impersonated" field and the JWT has the "support" field @alweber-cap.